config.c

```diff
@@ -662,16 +662,19 @@ int reload_config(struct mako_config *config, int argc, char **argv) {
 	struct mako_config new_config = {0};
 	init_default_config(&new_config);
 
-	if (load_config_file(&new_config) != 0) {
-		fprintf(stderr, "Failed to reload config\n");
+	int config_status = load_config_file(&new_config);
+	int args_status = parse_config_arguments(&new_config, argc, argv);
+
+	if (args_status > 0) {
```

Comment:
The check was `!=`. I'd prefer to keep it that way since it's a more idiomatic way to check for errors in C.

Reply:
Ah, I see. Okay, nevermind :P